apps/guardian-ui/src/utils/api.ts

```diff
@@ -15,6 +16,26 @@ export function getModuleParamsFromConfig<T extends AnyModuleParams[0]>(
   return module ? module[1] : null;
 }
 
+/**
+ * Given a config, filter out all non-default modules
+ */
+export function getOtherModuleParamsFromConfig(
+  config: ConfigGenParams | null
+): object {
+  if (!config) return {};
+
+  return Object.keys(config.modules)
+    .filter(
+      (key) =>
+        config.modules[parseInt(key)][0] != ModuleKind.Mint &&
+        config.modules[parseInt(key)][0] !== ModuleKind.Ln &&
+        config.modules[parseInt(key)][0] !== ModuleKind.Wallet
+    )
+    .reduce((cur, key) => {
+      return Object.assign(cur, { [key]: config.modules[parseInt(key)] });
+    }, {});
+}
+
```

Comment:
Notice the `parseInt` calls here. Seems like our types are kinda wrong? I would expect to be able to just index into an object with whatever keys it has.

Reply:
Interfaces in TypeScript can be subsets of the actual underlying object, therefore any call to `Object.keys` or other key-grabbing functions will always loosen the type to `string`.

I'll have to check the way the API serializes it, but I think I originally saw the keys being numeric and typed them to number, but maybe we should just treat them like strings since object keys are always strings.